src/lib/parquet/src/Flow/Parquet/ParquetFile/Data/PlainValuesPacker.php

```diff
@@ -50,35 +36,33 @@ public function packValues(FlatColumn $column, array $values) : void
                 switch ($column->logicalType()?->name()) {
                     case LogicalType::TIME:
                     case LogicalType::TIMESTAMP:
-                        $this->writer->writeInts64($parquetValues);
-
-                        break;
                     case null:
-                        $this->writer->writeInts64($parquetValues);
+                        $this->writer->writeInts64($values);
+                        $this->writer->writeInts64($values);
```

Comment:
This call is duplicated, is that correct?

Reply:
🤦‍♂️ thanks!